frontend/src/components/create-contract/CreateContractForm.tsx

```diff
@@ -6,99 +6,197 @@ import { ListBox } from '../ListBox';
 import { acceptanceTestsItems, identityItems, labelItems } from './labels';
 import OptionsSummary from './OptionsSummary';
 import PaymentSummary from './PaymentSummary';
+import { Formik, Form } from 'formik';
+import React, { useState } from 'react';
+import { useSmartContracts } from 'components/smart-contracts/useSmartContracts';
+import { ApproveERC20Modal } from 'components/smart-contracts/ApproveERC20Modal';
+import { PostJobModal } from 'components/smart-contracts/PostJobModal';
+import { CreateFormValues } from 'components/forms/types';
+import { useHistory } from 'react-router-dom';
 
 const CreateContractForm = () => {
+  const initialValues: CreateFormValues = {
+    title: '',
+    description: '',
+    'acceptance-criteria': '',
+    bounty: '',
+  };
+
+  const history = useHistory();
+
+  const { contracts } = useSmartContracts();
+  const [submitting, setSubmitting] = useState(false);
+  const [formData, setValidatedFormData] =
+    useState<CreateFormValues>(initialValues);
```

Comment:
Correct me if I'm wrong, but I don't think this is necessary with Formik. You can use their `validation` method to validate the form data, and then it should just be available as `values` in `onSubmit`